spec/models/item_spec.rb

```diff
@@ -0,0 +1,12 @@
+describe Item do
+	before(:each) do
+		FactoryGirl.create(:item)
+	end
+
+  describe "#price" do
+    it "returns the price with two decimal places of precision" do
+			item = Item.last
+			expect(item.price).to eq("149.00")
```

Comment:
I've taken a look at a lot of these "tab detected" issues and can't find anything out of whack.